odl/discr/grid.py

```diff
@@ -93,29 +83,23 @@ class TensorGrid(Set):
     This is a sparse representation of a collection of n-dimensional
     points defined by the tensor product of n coordinate vectors.
 
-    Example:
-    ``x = (x_1, x_2), y = (y_1, y_2, y_3, y_4), z = (z_1, z_2, z_3)``
+    Example::
+
+        x = (x1, x2), y = (y1, y2, y3, y4), z = (z1, z2, z3)
+
     The resulting grid consists of all possible combinations
```

Comment:
I'll rewrite that slightly, but the next lines show the ordering. I'll make it more explicit.